packages/salesforcedx-vscode-apex-debugger/src/index.ts

```diff
@@ -86,7 +98,150 @@ function registerCommands(): vscode.Disposable {
       }
     }
   );
-  return vscode.Disposable.from(customEventHandler);
+  const exceptionBreakpointCmd = vscode.commands.registerCommand(
+    'sfdx.debug.exception.breakpoint',
+    configureExceptionBreakpoint
+  );
+  const startSessionHandler = vscode.debug.onDidStartDebugSession(session => {
+    cachedExceptionBreakpoints.forEach(breakpoint => {
+      const args: SetExceptionBreakpointsArguments = {
+        exceptionInfo: breakpoint
+      };
+      session.customRequest(EXCEPTION_BREAKPOINT_REQUEST, args);
```

Comment:
When a debugger session starts, add all exception breakpoints that were selected while without a debugger session.